ids/format_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ids
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFormat(t *testing.T) {
+	type (
+		idInterface interface {
+			String() string
+			Hex() string
+		}
```

Comment:
This seems to be duplicated in the test and in the code, should we define this interface in the code and use it in both places? Perhaps `type formatted interface` or something

Reply:
Whoops!